packages/plugin-pre/src/PreDomParser.ts

```diff
@@ -0,0 +1,21 @@
+import { PreNode } from './PreNode';
+import { AbstractParser } from '../../plugin-parser/src/AbstractParser';
+import { DomParsingEngine } from '../../plugin-dom/src/DomParsingEngine';
+
+export class PreDomParser extends AbstractParser<Node> {
+    static id = 'dom';
+    engine: DomParsingEngine;
+
+    predicate = (item: Node): boolean => {
+        return item instanceof Element && item.tagName === 'PRE';
+    };
+
+    async parse(item: Element): Promise<PreNode[]> {
+        const pre = new PreNode();
+        pre.attributes = this.engine.parseAttributes(item);
+        const children = await this.engine.parse(...item.childNodes);
+        // TODO: parse contained \n as LineBreaks.
```

Comment:
So, in my branch, you could use `new this.engine.editor.configuration.defaults.Separator()` to create a new "separator" using the editor configuration.

Reply:
https://github.com/odoo-dev/jabberwock/pull/162/commits/54c05ed308fde232bf092c3be2d0b1f7d8ed85fa